content/doc/book/blueocean/index.adoc

```diff
@@ -110,68 +124,87 @@ allowfullscreen></iframe>
-with any Freestyle jobs that you have configured in your system. However, they
-won't be able to benefit from any of the features built for Pipelines – for
+with any freestyle jobs you already have configured on your Jenkins instance.
+However, you will not benefit from any of the features built for Pipelines – for
 example, Pipeline visualization.
 
-As Blue Ocean is designed to be extensible it will be possible for the
-community to extend it for other job types in the future.
+As Blue Ocean is designed to be extensible, it is possible for the Jenkins
+community to extend Blue Ocean to support other job types in the future.
+
 
-=== What does this mean for the classic Jenkins UI?
+=== What does this mean for the Jenkins classic UI?
 
-The intention is that as Blue Ocean matures there will be less and less reasons
-for users to go back to the existing UI.
+The intention is that as Blue Ocean matures, there will be fewer reasons for
+users to go back to the existing "classic UI". Read more about the classic UI
+in link:../pipeline/getting-started[Getting started with Pipeline].
 
-For example, in the first version we will mainly be targeting Pipeline jobs.
+For example, early versions of Blue Ocean are mainly targeted at Pipeline jobs.
 You might be able to see your existing non-pipeline jobs in Blue Ocean but it
-might not be possible to configure them from the new UI for some time. This
-means users will have to jump back to the classic UI for configuration of
-non-pipeline jobs.
+might not be possible to configure them from the Blue Ocean UI for some time.
+This means users will have to jump back to the classic UI to configure
+items/projects/jobs other than Pipeline ones.
+
+There are likely going to be more examples of this, which is why the classic UI
+will remain important in the long term.
 
-There are likely going to be more examples of this and that's why the classic
-UI will still be important in the long term.
 
 === What does this mean for my plugins?
 
-Extensibility is a pretty core concept to Jenkins, so being able to extend the
-Blue Ocean UI is important. Based on some research, we worked out a way to
-allow `<ExtensionPoint name=..>` to be used in the markup of Blue Ocean,
-leaving places for plugins to contribute to the UI (plugins can have their own
-Blue Ocean extension points, just like they do today in Jenkins). Blue Ocean
-itself (as it is so far) is implemented using these extension points.
+Extensibility is a core feature of Jenkins. Therefore, being able to extend the
+Blue Ocean UI is important. The +
+`<ExtensionPoint name=..>` can be used in the markup of Blue Ocean, leaving
+places for plugins to contribute to the Blue Ocean UI - i.e. plugins can have
+their own Blue Ocean extension points, just like they can in the Jenkins classic
+UI. So far, Blue Ocean itself is implemented using these extension points.
+
 Extensions are delivered by plugins, as normal, only if they wish to contribute
```

Comment:
> only if they wish

Who is "they"? Are "they" meant to be the extensions or the plugins?

Reply:
@gilesgas Think plugins.